### DISCUSSION TITLE
Inverted accuracy and accuracy range fail to go with it.

### DISCUSSION THREAD

**Discussion by Redageddon** [ppy/osu#19618] [Ideas]
It could be a purely visual mod or something that instead of showing your current accuracy, it shows the maximum possible accuracy that you can get if you get perfect hits for the rest of the map. So say for example if you normally fail the first note of a map it will say 0% accuracy, but with this "inverted accuracy" it would show something like 99.XX. 

If this was implemented, then there could also be an accuracy fail mod, kind of like a middle ground of an instant fail. What you could do is set a minimum achievable percentage needed that you want, and if the inverted accuracy drops beneath that, you instantly fail. (this is because inverted accuracy would always go down, and can never go up, so you can guarantee an ability to just restart the map after that point)

**Comment by peppy** [comment]
This has been proposed before I think. https://github.com/ppy/osu/issues/9336 touches on it.

**Reply by Redageddon** [reply]
Ah I see, yah i wasn't able to find that because of a terrible title name. But it also doesn't go over the inverted accuracy fail mod
